lib/dor/assembly/content_metadata.rb

```diff
@@ -14,29 +12,26 @@ class ContentMetadata
       #   :druid = required - a string of druid of the repository object's druid id (with or without 'druid:' prefix)
       #   :objects = required - an array of Assembly::ObjectFile objects containing the list of files to add to content metadata
       #                NOTE: if you set the :bundle option to :prebundled, you will need to pass in an array of arrays, and not a flat array, as noted below
-      #   :style = optional - a symbol containing the style of metadata to create, allowed values are
-      #                 :simple_image (default), contentMetadata type="image", resource type="image"
-      #                 :file, contentMetadata type="file", resource type="file"
-      #                 :simple_book, contentMetadata type="book", resource type="page", but any resource which has file(s) other than an image, and also contains no images at all, will be resource type="object"
-      #                 :book_with_pdf, contentMetadata type="book", resource type="page", but any resource which has any file(s) other than an image will be resource type="object" - NOTE: THIS IS DEPRECATED
-      #                 :book_as_image, as simple_book, but with contentMetadata type="book", resource type="image" (same rule applies for resources with non images)  - NOTE: THIS IS DEPRECATED
-      #                 :map, like simple_image, but with contentMetadata type="map", resource type="image"
-      #                 :3d, contentMetadata type="3d", ".obj" and other configured 3d extension files go into resource_type="3d", everything else into resource_type="file"
-      #                 :webarchive-seed, contentMetadata type="webarchive-seed", resource type="image"
```

Comment:
this code is still used for :webarchive-seed, yes?  probably crawls also

Reply:
No, this is only for stubMetadata stuff from Goobi.